tests/api-resources/test_resources.py

```diff
@@ -228,7 +245,7 @@ def test_create_publish_new_revision(client, minimal_record,
     )
 
     assert response.status_code == 201
-    assert response.json['revision_id'] == 1
+    assert response.json['revision_id'] == 4
```

Comment:
Question: why is this changed? Seems strange that this would need to change if it has previously passed.

Reply:
This changed due to the draft being now soft-deleted instead (of hard-deleting). So then when creating a new revision the previously deleted draft is used instead of creating a new one.